crates/scroll/payload/src/builder.rs

```diff
@@ -382,14 +394,10 @@ where
         builder_config: &ScrollBuilderConfig,
     ) -> Result<impl BlockBuilder<Primitives = Evm::Primitives> + 'a, PayloadBuilderError> {
         // get the base fee for the attributes.
-        let base_fee: u64 = if self.chain_spec.is_curie_active_at_block(self.parent().number + 1) {
```

Comment:
I guess this means we don't support building pre-Curie blocks anymore? (Which is fine.)